keys.go

```diff
@@ -130,24 +137,48 @@ func (c *Client) CreateRootKey(ctx context.Context, name string, expiration *tim
 	return c.CreateKey(ctx, name, expiration, false)
 }
 
+// CreateRootKeyWithSHA1 creates a new, non-extractable key resource without
+// key material using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateRootKeyWithSHA1(ctx context.Context, name string, expiration *time.Time) (*Key, error) {
+	return c.CreateKeyWithSHA1(ctx, name, expiration, false)
+}
+
 // CreateStandardKey creates a new, extractable key resource without
 // key material.
 func (c *Client) CreateStandardKey(ctx context.Context, name string, expiration *time.Time) (*Key, error) {
 	return c.CreateKey(ctx, name, expiration, true)
 }
 
+// CreateStandardKeyWithSHA1 creates a new, extractable key resource without
+// key material using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateStandardKeyWithSHA1(ctx context.Context, name string, expiration *time.Time) (*Key, error) {
+	return c.CreateKeyWithSHA1(ctx, name, expiration, true)
+}
+
 // CreateImportedRootKey creates a new, non-extractable key resource
 // with the given key material.
 func (c *Client) CreateImportedRootKey(ctx context.Context, name string, expiration *time.Time, payload, encryptedNonce, iv string) (*Key, error) {
 	return c.CreateImportedKey(ctx, name, expiration, payload, encryptedNonce, iv, false)
 }
 
+// CreateImportedRootKeyWithSHA1 creates a new, non-extractable key resource
+// with the given key material and using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateImportedRootKeyWithSHA1(ctx context.Context, name string, expiration *time.Time, payload, encryptedNonce, iv string) (*Key, error) {
+	return c.CreateImportedKeyWithSHA1(ctx, name, expiration, payload, encryptedNonce, iv, false)
+}
+
```

Comment:
Pretty sure you can leave this out, as it is exactly the same as CreateImportedKeyWithSHA1.

The only reason the `CreateImportedRootKey` method exists is for backwards compatibility. It was there previously and I did not want to remove it.

Reply:
Done.